src/main/java/frc/robot/commands/ShooterTargetCommand.java

```diff
@@ -18,105 +18,78 @@
 
 public class ShooterTargetCommand extends CommandBase {
 
-  HubTracking hubTracking;
-  Shooter shooter;
-  HubData data;
-  double range = 0;
-  boolean dataCollected = false;
-
-  InterpolatorTable flywheelTable = new InterpolatorTable(
+  
+  public static InterpolatorTable flywheelTable = new InterpolatorTable(
```

Comment:
one thing I don't like about this approach is that the interpolator table has to exist in 2 places now ... a longer term solution to that would be great